Resources/translations/ezrepoforms_content_type.en.xlf

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+  <file source-language="en" target-language="en" datatype="plaintext" original="file.ext">
+    <header>
+      <tool tool-id="symfony" tool-name="Symfony"/>
+    </header>
+    <body>
+      <trans-unit id="972127730f863287531a351da529a802" resname="content_type.edit_title">
+        <source>content_type.edit_title</source>
+        <target>__content_type.edit_title</target>
```

Comment:
Right. Then it may be ok to keep it as is, so translators don't waste their time. Ideally we shouldn't have duplicate entries like this, but if the script generates them, it's easier to just keep them...